index.js

```diff
@@ -1,17 +1,60 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function(a, b){return a-b});
+    this.length =+1;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if(this.items.includes(this.items[pos])){
```

Comment:
your includes() is an extra step from what you need, just with your return statement it'll be able to find the value that it's looking for.

Reply:
What you're doing in this function is for it to return a certain value from the index it has as an argument; your error handling tells you that if this value is not found in the length, it will throw an error. So that leaves the margin of numbers within your length! That for sure tells you that this index exists within your array; all that you need is to call it (which is what you do in your return statement).